### DISCUSSION TITLE
Why the start-up process of zephry is so long?

### DISCUSSION THREAD

**Discussion by sudashannon** [zephyrproject-rtos/zephyr#40284] [General]
Forgive me for my title.
I just want to use a new embedded RTOS. Should I install a lot of python modules?
On my opinion, the build framework of zephyr is too fat to use.

**Comment by sudashannon** [comment]
- An user defined toolchain and build framework
- A module-configured method
- All code

That is what I need.

**Comment by carlescufi** [comment]
>I just want to use a new embedded RTOS. Should I install a lot of python modules?

There are several types of RTOS. If all you want is a simple kernel with no build system then there are other options available. Zephyr is a complex project, so it needs a complete build and configuration system, and Python is used for parts of that.

**Comment by carlescufi** [comment]
I will convert this to a discussion in the meantime.
